### PR TITLE
Add subclass as a POI property

### PR DESCRIPTION
**Pull request by jbgriesner**
Simply add the POI subclass in the geocoder answer.
unwrap clean coming soon...

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -41,7 +41,20 @@ fn build_poi_properties(row: &Row) -> Result<Vec<Property>, String> {
...
             value: v.unwrap_or("".to_string()),
         })
-        .collect())
+        .collect::<Vec<Property>>();
+
+    let subclass = row.get_opt("subclass")
+        .unwrap()
+        .map_err(|e| warn!("impossible to get subclass for {} because {}", name, e))
+        .ok()
+        .unwrap();
```
Comment (antoine-de) [inline src/main.rs]: we don't want to crash there

File: src/main.rs [hunk 2]
```diff
@@ -41,7 +41,20 @@ fn build_poi_properties(row: &Row) -> Result<Vec<Property>, String> {
             key: k,
             value: v.unwrap_or("".to_string()),
         })
-        .collect())
+        .collect::<Vec<Property>>();
+
+    let subclass = row.get_opt("subclass")
```
Comment (antoine-de) [inline src/main.rs]: I don't feel that subclass is a good name

Comment (jbgriesner) [reply src/main.rs]: what about poi_subclass ?

File: src/main.rs (hunk 1 above)
Comment (jbgriesner) [reply src/main.rs]: done.

**Pull request merged**